e2e-tests/demand-backup-encrypted/run

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
we have `demand-backup-encrypted-with-tls`, I think we can safely delete this `demand-backup-encrypted`, it is absolutely the same